scripts/03_fetch_contacts.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ $# -ne 5 ]; then
+    echo "Usage: sh 04_fetch_contacts.sh <user> <rapid-pro-root> <rapid-pro-server> <rapid-pro-token> <data-root>"
+    echo "Downloads all contacts from Rapid Pro"
+    exit
+fi
+
+USER=$1
+RP_DIR=$2
+RP_SERVER=$3
+RP_TOKEN=$4
+DATA_ROOT=$5
+
+TEST_CONTACTS_PATH="/home/princelysid/projects/AVF/maap_somalia/data/test_contacts.json"
+
+cd "$RP_DIR/fetch_contacts"
+
+mkdir -p "$DATA_ROOT/03 Raw Contacts"
```

Comment:
This index (03 Raw Contacts) mis-matches that in fetch_contacts (04_fetch_contacts.sh). I recognise that this is a bit of a pain point, but for now we're trying to keep the script indices in sync with the data indices to help us tell which data came from which stage